source/install/cloud-changelog.md

```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-02-02](#release-2023-02-02)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
-- [Release 2022-11-17](#release-2022-11-17)
+
+## Release 2023-02-02
+
+### Highlights
+
+#### Annual Cloud Subscriptions
+ - On the purchase modal, admins are now able to buy an annual cloud subscription for a user count that exceeds the current user count.
```

Comment:
@cwarnermm made a suggestion below but may need your help on this rewording. It should not say for a user count that exceeds their current user count - it could be the same or more.

```suggestion
 - On the purchase modal, admins are now able to buy an annual cloud subscription starting from their current user count .
```